github/pkg/adapter/adapter_test.go

```diff
@@ -18,27 +18,29 @@ package adapter
 
 import (
 	"bytes"
-	"context"
-	"errors"
+	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"strconv"
-	"strings"
 	"testing"
+	"time"
 
 	"knative.dev/eventing-contrib/github/pkg/apis/sources/v1alpha1"
```

Comment:
Let's group this import together with the other third-party packages. Or even better: group all `knative.dev` imports together.

Reply:
Done.